core/rint/src/TRint.cxx

```diff
@@ -498,7 +498,6 @@ void TRint::Run(Bool_t retrn)
    }
 
    if (QuitOpt()) {
-      printf("\n");
```

Comment:
For better or worse, this 'can' be noticed by users (eg. that have some automation of sorts that includes diff or what not).  So we do (yes, sigh ... :) ) need to explicitly announce it in the release notes (I vaguely remember at least one issue related to this kind of empty line).

Reply:
Ok why not, but who would do diffs of running `root` with not input files and no interactivity :smiley: We do it to implicitly create modules, but that's probably it.

Maybe some people use `root -b -q` to get the banner to validate the ROOT version...

Reply:
Thanks for adding the release notes :)

Reply:
I still can find the exact report.  I am slightly remembering that the user/experiment was needing to make a distinction between `root -b -l -q` returning absolutely nothing and `root.exe -b -l -q` having that extra line ... can't yet remember the exact context.

Reply:
and I might be confused with other (similar but unrelated discussion like https://github.com/root-project/cling/issues/61#issuecomment-256393398)